jenkins/dev_env/build_rpm.sh

```diff
@@ -38,9 +38,6 @@ ls /var/lib/ha_env/ | grep yum_init || {
     echo "Please complete preqs."
 }
 
-req_file=${REPO_PATH}/jenkins/pyinstaller/v2/requirements.txt
-python3 -m pip install -r $req_file
```

Comment:
since we are deleting the files , we need to remove the "excludes" from whiltesource.config
Can you please ask Swanad gadre to start a PR for that once this PR is merged ?

Reply:
Also please confirm that ./ha/pcswrap/Makefile is not being used anywhere

Reply:
yes, I can, build install, and deploy all works well didn't get any error.
requirements.txt is used in /root/cortx-ha/ha/pcswrap/Makefile but I checked the makefile is not getting used in Kubernetes build environment.